block-storage/src/main/scala/coop/rchain/blockstorage/dag/Finalizer.scala

```diff
@@ -130,7 +131,7 @@ final case class Finalizer[M, S](msgMap: Map[M, Message[M, S]]) {
     // Total stake
     val totalStake = bondsMap.values.toSeq.sum
     // Calculate if 2/3 of stake supporting next layer
-    fullPartitionStake.toDouble / totalStake > 2d / 3
+    Stake.prevails(fullPartitionStake, totalStake)
```

Comment:
In previous version *greater* (`>`) comparsion operator and floating-point calculations were used . More correct is to use *greater or equal* (`>=`) operator and integer calculations.

Reply:
I was wrong. As shown in the article [Byzantine_fault](https://en.wikipedia.org/wiki/Byzantine_fault) (I was guided by the Russian version), the stake must be **strictly greater** than 2/3.